src/actions/clusterActions.js

```diff
@@ -166,7 +167,7 @@ export function clustersLoad() {
     });
 
     const lastUpdated = Date.now();
-    dispatch(clustersLoadSuccessV4(v4ClustersObject, lastUpdated));
+    // dispatch(clustersLoadSuccessV4(v4ClustersObject, lastUpdated));
```

Comment:
Would like to avoid commented out code in the codebase.

Reply:
sorry... I will end up writing a git hook for this because I am doing it so frequently!